src/App.vue

```diff
@@ -765,6 +765,11 @@ export default {
 			height: 100vh !important;
 		}
 	}
+
+	// TODO remove after https://github.com/nextcloud-libraries/nextcloud-vue/pull/4760 regression is fixed
```

Comment:
```suggestion
	// TODO remove after https://github.com/nextcloud-libraries/nextcloud-vue/issues/4896 regression is fixed
```

PS you could just send this PR upstream?